From ce2c13a41240a6638acdb3f2af1604c0b1c19c37 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Wed, 25 Nov 2020 15:57:35 +0200
Subject: [PATCH 05/78] drivers: dwmac-s32cc: Correct TX clock name

Use the same clock names for TX in GMAC driver and DTS.

Issue: ALB-5929
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-s32cc.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-s32cc.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-s32cc.c
index f93af585221f..d32decc6767e 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-s32cc.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-s32cc.c
@@ -161,17 +161,17 @@ static int s32cc_dwmac_probe(struct platform_device *pdev)
 
 	switch (gmac->intf_mode) {
 	case PHY_INTERFACE_MODE_SGMII:
-		tx_clk = "sgmii_tx";
+		tx_clk = "tx_sgmii";
 		break;
 	case PHY_INTERFACE_MODE_RGMII:
-		tx_clk = "rgmii_tx";
+		tx_clk = "tx_rgmii";
 		break;
 	case PHY_INTERFACE_MODE_RMII:
-		tx_clk = "rmii_tx";
+		tx_clk = "tx_rmii";
 		break;
 	default:
 	case PHY_INTERFACE_MODE_MII:
-		tx_clk = "mii_tx";
+		tx_clk = "tx_mii";
 		break;
 	};
 
-- 
2.25.1

